### PR TITLE
Some work on .pylintrc

### PR DESCRIPTION
**Pull request by whirm**

### CODE REVIEW

**Comment by whirm** [comment]
I'll wait for some feedback before merging

File: .pylintrc [hunk 1]
```diff
@@ -135,10 +139,13 @@ generated-members=
...
 # install python-enchant package.
-spelling-dict=
+spelling-dict=en
+
+# To install required dictionary:
+# sudo apt install aspell-en
 
 # List of comma separated words that should not be checked.
-spelling-ignore-words=
+spelling-ignore-words=Tribler,dispersy
```
Comment (lfdversluis) [inline .pylintrc]: pymdht?

File: .pylintrc [hunk 2]
```diff
@@ -292,7 +299,7 @@ no-docstring-rgx=^_
 
 # Minimum line length for functions/classes that require docstrings, shorter
 # ones are exempt.
-docstring-min-length=-1
+docstring-min-length=5
```
Comment (lfdversluis) [inline .pylintrc]: Why 5?

Comment (whirm) [inline .pylintrc]: Do you have a better number? :D

Comment (lfdversluis) [inline .pylintrc]: 3!

Comment (whirm) [inline .pylintrc]: Bingo!

**Pull request merged**